### ISSUE TITLE
nvda does not announce typed characters when creating a new folder 

### ISSUE THREAD

**Issue by fernando-jose-silva** [nvaccess/nvda#12120]
### Steps to reproduce:
turn on your computer, enter a folder as documents, open the menu bar, walk to the new folder and press enter
Now type the characters for the new folder name
also navigate with right and left arrows by the name entered 
### Actual behavior:
nvda does not say characters as you type.
  nvda does not inform the characters typed when navigating with left and right arrows 
### Expected behavior:
nvda must announce the characters as I type them and must also announce the characters already typed 
comments

this occurs during the first time the explorer is opened, example:
I turn on the computer and create the folder the problem occurs.
then I create another folder the problem does not occur.
restart the explorer process using the task manager, create a folder, the problem occurs. 
#### NVDA installed/portable/running from source:
instaled
#### NVDA version:
nvda.exe, NVDA alpha-21810,db664bef

#### Windows version:
10.0.19043.844
#### Does the issue still occur after restarting your computer?
yes
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
no
#### If add-ons are disabled, is your problem still occurring?
yes
#### Did you try to run the COM registry fixing tool in NVDA menu / tools?
no
log:
[explorer.txt](https://github.com/nvaccess/nvda/files/6085684/explorer.txt)

**Comment by XLTechie** [comment]
Can you please run the Com Reg Fixing tool (NVDA+n, t, r)? Some of these 
problems you have posted about recently seem potentially related.

**Comment by fernando-jose-silva** [comment]
thank you very much, solved this problem, now nvda read the characters when typing and browsing when creating a folder.
just as a record, I had to run the tool to run the registry correction tool 2 times
the first time i ran the tool and restarted windows.
It got worse, the explorer started to crash, even with the narrator.
I ran the tool again and restarted and the problem resolved.

**Comment by XLTechie** [comment]
It sounds like you might have some kind of deeper problem, if Explorer was 
crashing, even with Narrator.
A Windows repair might be in order.

Please close this issue if it is solved.
